PerimeterXModule/Internals/PxContext.cs

```diff
@@ -209,11 +210,14 @@ private bool shouldMonitorRequest(String uri, PxModuleConfigurationSection pxCon
 				}
 				return true;
 			}
-			else
-			{
-				return pxConfiguration.MonitorMode;
-			} 
-           
+			
+			if (!string.IsNullOrEmpty(pxConfiguration.ByPassMonitorHeader)) {
```

Comment:
the code style convention in C# (unfortunately) is to put the opening brackets in a new line

Reply:
good catch